Makefile

```diff
@@ -52,12 +52,16 @@ ocid: $(GO_SRC) | ${OCID_LINK}
 ocic: $(GO_SRC) | ${OCID_LINK}
 	$(GO) build -o $@ ./cmd/client/
 
+kpod: $(GO_SRC)
```

Comment:
do we need `${OCID_LINK}` as well? that's needed in order to work with the vendored code